kratos/tests/cpp_tests/sources/test_chunk.cpp

```diff
@@ -30,25 +30,76 @@ namespace Kratos {
 		{
 			int max_threads = OpenMPUtils::GetCurrentNumberOfThreads();
 
+			std::size_t block_size_in_bytes = 5; // the aligned block size is 8
+			std::size_t header_size = 2 * max_threads * sizeof(Chunk::SizeType);
+			std::size_t chunk_size_in_bytes = header_size + 1024;
+
+			Chunk chunk(block_size_in_bytes, chunk_size_in_bytes);
+			chunk.Initialize();
+
+			std::size_t block_size_after_alignment = 8;
+			std::size_t available_blocks_should_be = (chunk_size_in_bytes - 2* max_threads * sizeof(Chunk::SizeType)) / block_size_after_alignment;
+			KRATOS_CHECK_EQUAL(chunk.GetNumberOfAvailableBlocks(), available_blocks_should_be) << " Available block :" << chunk.GetNumberOfAvailableBlocks() << " vs " << available_blocks_should_be;
+		}
+
+		KRATOS_TEST_CASE_IN_SUITE(ChunkInitializeSmallBlock, KratosCoreFastSuite)
+		{
+			int max_threads = OpenMPUtils::GetCurrentNumberOfThreads();
```

Comment:
shouldn't this also be `GetNumThreads`?

Reply:
Not for this case. This is a bit misleading, that's why I split into `Initialize` and `InitializeParallel`. In this case the chunk is being called by a single thread context, so the max_threads will be one, which is consistent with the definition of `GetCurrentNumberOfThreads()`. We could use one, but this is better if a user copy/pastes this piece of code, as internally the chunk uses the same call.

Reply:
ok!